snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/importer/Rf2EffectiveTimeSlice.java

```diff
@@ -172,6 +172,10 @@ public void registerDependencies(long componentId, LongSet dependencies) {
 		}
 	}
 	
+	public void removeDependency(String componentId) {
```

Comment:
Not sure the current naming and parameter list captures your intent correctly. Let's have a chat when you have time.